UPGRADE-3.x.md

```diff
@@ -1,6 +1,8 @@
 UPGRADE 3.x
 ===========
 
+Injecting the block id is deprecated and will be automatically set.
```

Comment:
What should the user do though? Leave an empty `<argument/>` ?

If yes please add :

```
Instead, provide an empty (null?) argument:

## Before
`<argument>your.block</argument>`

## After
`<argument/>`
```